pip_check_reqs/find_missing_reqs.py

```diff
@@ -46,7 +46,7 @@ def find_missing_reqs(
         package_files: list[str] = []
         for item in package.files or []:
```

Comment:
This `for` is redundant now, we're just appending `str(relative_item_location)` to `package_files`, `len(package.files)` times.

This also smells like we're missing something about `package.files` here.